test/setup.js

```diff
@@ -91,21 +90,81 @@ function screenshotCreateOrCompare (app, t, name) {
   })
 }
 
+// Compares two PNGs, ignoring any transparent regions in bufExpected.
+// Returns true if they match.
+function compareIgnoringTransparency (bufActual, bufExpected) {
```

Comment:
I did it that way initially, but I like this better because it means we can use transparency a lot less. That makes it easier to update the screenshots when we change the CSS.

Also, I was first trying to avoid `config.IS_TEST` completely, but then I ended up needing it anyway to mock `cast.js`